python/sqlflow_submitter/env.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
```

Comment:
How about changing the filename into `seeding.py`?

Reply:
Done.